storage/queue.go

```diff
@@ -511,11 +521,19 @@ func (bq *baseQueue) processReplica(repl *Replica, clock *hlc.Clock) error {
 
 	log.VEventf(3, bq.ctx, "processing")
 	start := timeutil.Now()
-	if err := bq.impl.process(ctx, clock.Now(), repl, cfg); err != nil {
+	err := bq.impl.process(ctx, clock.Now(), repl, cfg)
+	duration := timeutil.Since(start)
+	log.VEventf(2, bq.ctx, "done (err=%s): %s", err, duration)
```

Comment:
also there's `log.ErrEvent{,f}` and we should probably get in the habit of using it whenever `err` or `pErr` is involved.